app/views/responses/form.html.erb

```diff
@@ -1,11 +1,10 @@
 <% content_for(:per_page_js, javascript_google_maps) %>
 <% @title_args = {id: @response.decorate.shortcode} %>
 
-<%# This needs to be up here so it runs before the MediaUploaderViews are initialized. %>
+<%# This needs to be up here so it runs before the FileUploaderViews are initialized. %>
 <%= javascript_doc_ready do %>
   ELMO.mediaUploaderManager = new ELMO.Views.MediaUploaderManager(<%=json(
-    el: ".response_form",
-    preview_template: render("responses/answers/dropzone_preview")
+    el: ".response_form"
```

Comment:
We can get rid of the ERB and `json` call now right?

Reply:
Can also change this please to `response-form` (dash). Both are present on the form (it's complicated, but underscores in CSS classes are deprecated).